src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -91,28 +94,31 @@ const ActiveState = ({
                         return (
                             <ActivityItem
                                 key={item.type + item.id}
-                                className="bcs-activity-feed-comment"
-                                data-testid="comment"
+                                className={hasReplies ? 'bcs-activity-thread' : 'bcs-activity-feed-comment'}
+                                data-testid={hasReplies ? 'activity-thread' : 'comment'}
```

Comment:
Should this be on the `ActivityThread` component, instead?

Reply:
I ask because ActivityItem already has a `data-testid` that we can use. If we want to select on ActivityThread, as well, we can add a `data-testid` there.

Reply:
No I get it 😄 Sounds good - fixed